src/main/g8/src/main/resources/local-shared.conf

```diff
@@ -2,10 +2,8 @@ $name$.grpc.interface = "127.0.0.1"
 akka.remote.artery.canonical.hostname = "127.0.0.1"
 akka.management.http.hostname = "127.0.0.1"
 
-## The following setting overrides for Akka Management, Akka Discovery and Akka Projections
-## should only be used in a local environment during development.
 akka.management.cluster.bootstrap.contact-point-discovery {
-  service-name = "$name;format="word-only,lowercase"$"
```

Comment:
any specific reason for now enforcing it to be word-only and lowercase? 

In any case, I think it's ok here to just use whatever the user have chosen, but maybe not for the DB url, user and password bellow. Then we also need to adapt the docker-compose file.

Reply:
I did a naive search and replace, I'll improve each usage.

Reply:
Hmm, in this particular change and on the DB-related usages, we seem to be using [`normalize`](http://www.foundweekends.org/giter8/formatting.html). I'll improve it to enforce normalized input (lower case and hyphenated).